static/static_test.go

```diff
@@ -0,0 +1,129 @@
+package static
```

Comment:
I tried to get as much of the interesting logic as I could easy test covered, but it's not comprehensive. If the handlers don't work right, it's going to be quite obvious from failing e2e tests.